tests/testlib/s2n_testlib.h

```diff
@@ -130,6 +130,12 @@ int s2n_connection_set_all_protocol_versions(struct s2n_connection *conn, uint8_
 #define S2N_ONE_TRAILING_BYTE_CERT_BIN         "../pems/one_trailing_byte_cert.bin"
 #define S2N_TWO_TRAILING_BYTE_CERT_BIN         "../pems/two_trailing_byte_cert.bin"
 
+/* This is a certificate chain with a SHA-1 legacy algorithm on the root and 
+ * a SHA-256 signature algorithm on the intermediate certificate. This is used to prove
+ * that our certificate validation code does not fail a root certificate signed with SHA-1.
+ */
```

Comment:
I think this could be a little clearer:
```
/* This is a certificate chain with a legacy SHA-1 signature on the root certificate and 
 * a SHA-256 signature on the intermediate certificate. This is used to prove
 * that our certificate validation code does not fail a root certificate signed with SHA-1.
 */
```